puree/src/main/java/com/cookpad/puree/storage/EnhancedPureeStorage.java

```diff
@@ -0,0 +1,7 @@
+package com.cookpad.puree.storage;
+
+public abstract class EnhancedPureeStorage implements PureeStorage {
+
+    public void delete(String type, long ageMillis) {
```

Comment:
Ideally, yes this should be abstract to enforce the subclass to implement the method.
However, I'm trying to future-proof this class so that adding methods in this class will not break existing clients that extends `EnhancedPureeStorage`.